crawlers/moonstreamcrawlers/cli.py

```diff
@@ -207,8 +208,9 @@ def processing_order(raw_order: str) -> ProcessingOrder:
     parser_ethcrawler_blocks_sync.add_argument(
         "-t",
         "--transactions",
-        choices=["True", "False"],
-        default="True",
+        action="store_const",
```

Comment:
@kompotkot : Use [`action="store_true"`](https://docs.python.org/3/library/argparse.html#action).

If you want default to be `True`, use a flag like `--notransactions` or `--skip-transactions` with `action="store_true"`.

Reply:
Done